jobs/cjd_sample/src/main/java/com/google/samples/SearchQuickstart.java

```diff
@@ -0,0 +1,262 @@
+          .setRequisitionId("584632885")
+          .setLocations(Arrays.asList("Palo Alto, CA"))
+          .setJobTitle("Graphic Designer");
+
+  // Below job matches the job category of BUSINESS_OPERATIONS.
+  public static final Job JOB_SAMPLE_6 =
+      new Job()
+          .setApplicationInstruction("Apply to this job on www.careers.google.com. ")
+          .setDescription(
+              "The Business Strategy & Operations organization provides business critical insights "
+                  + "using analytics, ensures cross functional alignment of goals and execution, "
+                  + "and helps teams drive strategic partnerships and new initiatives forward. We "
+                  + "stay focused on aligning the highest-level company priorities with strong "
+                  + "day-to-day operations, and help evolve early stage ideas into future-growth "
+                  + "initiatives.")
+          .setRequisitionId("584632886")
+          .setLocations(Arrays.asList("Palo Alto, CA"))
+          .setJobTitle("Senior Business Analyst");
+
+  // Below job matches the job category of CLEANING_AND_FACILITIES.
+  public static final Job JOB_SAMPLE_7 =
+      new Job()
+          .setApplicationInstruction("Apply to this job on www.careers.google.com. ")
+          .setDescription(
+              "As a Room Attendant, your contribution helps ensure guests an enjoyable and "
+                  + "comfortable stay. As Housekeeper, you will Clean, dust, polish and vacuum "
+                  + "to make sure guest rooms and bathrooms meet standards.")
+          .setRequisitionId("584632887")
+          .setLocations(Arrays.asList("Palo Alto, CA"))
+          .setJobTitle("Housekeeping Room Attendant");
+
+  public static final List<Job> JOBS =
+      Arrays.asList(
+          JOB_SAMPLE_1,
+          JOB_SAMPLE_2,
+          JOB_SAMPLE_3,
+          JOB_SAMPLE_4,
+          JOB_SAMPLE_5,
+          JOB_SAMPLE_6,
+          JOB_SAMPLE_7);
+
+  /**
+   * Insert sample jobs with given companyName.
+   */
+  public static void insert(String companyName) throws IOException {
+    for (Job job : JOBS) {
+      job.setCompanyName(companyName);
+      Job returnJob =
+          jobService
+              .jobs()
+              .create(new CreateJobRequest().setDisableStreetAddressResolution(true).setJob(job))
+              .execute();
+      String jobName = returnJob.getName();
+      System.out.println(jobName);
+    }
+  }
+  //[END create_jobs]
+
+  //[START basic_search]
+
+  /**
+   * Simple search jobs with keyword.
+   */
+  public static void basicSearcJobs() throws IOException {
+    // Make sure to set the requestMetadata to reflect the unique jobb seeker. Obfuscate these fields.
+    RequestMetadata requestMetadata =
+        new RequestMetadata()
+            .setUserId("HashedUserId") // Make sure to hash the userID
+            .setSessionId("HashedSessionID") // Make sure to hash the sessionID
+            .setDomain(
```

Comment:
The parameter is on a newline, but the comment is also broken into multiple lines as well.

Pretty sure checkstyle is not going to like that comment indentation level as well, but double check to be sure.

Reply:
Done. Passed checkstyle.